rust/src/settings.rs

```diff
@@ -436,6 +437,42 @@ pub struct ModelSettings {
     pub size: usize,
 }
 
+#[derive(Debug, Deserialize)]
+/// Metrics settings.
+pub struct MetricsSettings {
+    /// The URL address of the metrics store.
+    ///
+    /// # Examples
+    ///
+    /// **TOML**
+    /// ```text
+    /// [metrics]
+    /// store_url = "http://localhost:8086"
+    /// ```
+    ///
+    /// **Environment variable**
+    /// ```text
+    /// XAYNET_API__STORE_URL=http://localhost:8086
+    /// ```
+    pub store_url: String,
+
+    /// The name of the metrics store.
+    ///
+    /// # Examples
+    ///
+    /// **TOML**
+    /// ```text
+    /// [metrics]
+    /// store_name = "test"
+    /// ```
+    ///
+    /// **Environment variable**
+    /// ```text
+    /// XAYNET_API__STORE_NAME=test
+    /// ```
+    pub store_name: String,
```

Comment:
I think it will be clearer to a user if we use the influxdb specific term `database_name` or `db_name` instead of `store_name`.

Reply:
yes, at the time I was erring on the side of being agnostic to the backend, but really these settings are influx-specific (as per @little-dude 's comment above). the naming should be better now together with the new nested `[metrics.influxdb]` setting.